dashboard/app/controllers/plc_enrollment_evaluations_controller.rb

```diff
@@ -0,0 +1,22 @@
+class PlcEnrollmentEvaluationsController < ApplicationController
```

Comment:
Long term - yes. There's a tech debt item to namespace the models and controllers.